### PR TITLE
tikv-control: make a parameter in right places

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Modify the parameter from db to data-dir

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9549 https://github.com/pingcap/docs/pull/9762
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-control.md [hunk 1]
```diff
@@ -282,26 +287,32 @@ middle_key_by_approximate_size:
...
-`compact` 命令可以对单个 TiKV 进行手动 compact。如果指定 `--from` 和 `--to` 选项，那么它们的参数也是 escaped raw key 形式的。
+`compact` 命令可以对单个 TiKV 进行手动 compact。
 
-- `--host` 参数可以指定要 compact 的 TiKV。
-- `-d` 参数可以指定要 compact 的 RocksDB，有 `kv` 和 `raft` 参数值可以选。
-- `--data-dir` 参数指定本地 TiKV 数据的目录路径。
-- `--threads` 参数可以指定 compact 的并发数，默认值是 8。一般来说，并发数越大，compact 的速度越快，但是也会对服务造成影响，所以需要根据情况选择合适的并发数。
-- `--bottommost` 参数可以指定 compact 是否包括最下层的文件。可选值为 `default`、`skip` 和 `force`，默认为 `default`。
+- `--from` 和 `--to` 选项以 escaped raw key 形式指定 compact 的范围。如果没有设置，表示 compact 整个 TiKV。
+- `region` 选项指定 compact Region 的范围。如果设置，则 `--from` 和 `--to` 选项会被忽略。
```
Comment (Connor1996) [inline tikv-control.md:293]:
```suggestion
- `--region` 选项指定 compact Region 的范围。如果设置，则 `--from` 和 `--to` 选项会被忽略。
```

File: tikv-control.md [hunk 2]
```diff
@@ -282,26 +287,32 @@ middle_key_by_approximate_size:
...
+- `--host` 选项可以指定要 compact 的 TiKV。
```
Comment (Connor1996) [inline tikv-control.md:294]: 删除，这是 tikv-ctl 通用参数，不是该命令的参数

File: tikv-control.md [hunk 3]
```diff
@@ -282,26 +287,32 @@ middle_key_by_approximate_size:
...
+- `--db` 选项可以指定要 compact 的 RocksDB，可选值为`kv` 和 `raft`。
+- `--data-dir` 选项指定本地 TiKV 数据的目录路径。
```
Comment (Connor1996) [inline tikv-control.md:296]: 同上

File: tikv-control.md [hunk 4]
```diff
@@ -282,26 +287,30 @@ middle_key_by_approximate_size:
...
+`compact` 命令可以对单个 TiKV 进行手动 compact。
 
-- `--host` 参数可以指定要 compact 的 TiKV。
-- `-d` 参数可以指定要 compact 的 RocksDB，有 `kv` 和 `raft` 参数值可以选。
-- `--data-dir` 参数指定本地 TiKV 数据的目录路径。
-- `--threads` 参数可以指定 compact 的并发数，默认值是 8。一般来说，并发数越大，compact 的速度越快，但是也会对服务造成影响，所以需要根据情况选择合适的并发数。
-- `--bottommost` 参数可以指定 compact 是否包括最下层的文件。可选值为 `default`、`skip` 和 `force`，默认为 `default`。
+- `--from` 和 `--to` 选项以 escaped raw key 形式指定 compact 的范围。如果没有设置，表示 compact 整个 TiKV。
+- `--region` 选项指定 compact Region 的范围。如果设置，则 `--from` 和 `--to` 选项会被忽略。
+- `--db` 选项可以指定要 compact 的 RocksDB，可选值为`kv` 和 `raft`。
```
Comment (TomShawn) [inline tikv-control.md:294]:
```suggestion
- `--db` 选项可以指定要 compact 的 RocksDB，可选值为 `kv` 和 `raft`。
```

File: tikv-control.md [hunk 5]
```diff
@@ -282,26 +287,30 @@ middle_key_by_approximate_size:
...
+    success!
+    ```
 
-```
-success!
-```
+- 在远程模式 compact data，执行如下命令：
+
+    ```shell
+    $ tikv-ctl --host ip:port compact --db kv
```
Comment (TomShawn) [inline tikv-control.md:311]:
```suggestion
    tikv-ctl --host ip:port compact --db kv
```

File: tikv-control.md [hunk 6]
```diff
@@ -282,26 +287,30 @@ middle_key_by_approximate_size:
...
 
-{{< copyable "shell-regular" >}}
+- 在本地模式 compact data，执行如下命令：
 
-```shell
-tikv-ctl --data-dir /path/to/tikv compact -d kv
-```
+    ```shell
+    tikv-ctl --data-dir /path/to/tikv compact --db kv
+    success!
```
Comment (TomShawn) [inline tikv-control.md:305]:
```suggestion
```

File: tikv-control.md [hunk 7]
```diff
@@ -282,26 +287,30 @@ middle_key_by_approximate_size:
...
+    ```
 
-```
-success!
-```
+- 在远程模式 compact data，执行如下命令：
+
+    ```shell
+    tikv-ctl --host ip:port compact --db kv
+    success!
```
Comment (TomShawn) [inline tikv-control.md:312]:
```suggestion
```

**Review by Connor1996 — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

File: tikv-control.md [hunk 8]
```diff
@@ -115,7 +115,12 @@ tikv-ctl 提供以下两种运行模式：
     store:"127.0.0.1:20160" compact db:KV cf:default range:([], []) success!
     ```
 
-- **本地模式**。通过 `--data-dir` 选项来指定本地 TiKV 数据的目录路径。在此模式下，需要停止正在运行的 TiKV 实例。
+- **本地模式**：
+
+    - 通过 `--data-dir` 选项来指定本地 TiKV 数据的目录路径。
+    - 通过 `--config` 选项来指定本地 TiKV 配置文件到路径。
+
+    在此模式下，需要停止正在运行的 TiKV 实例。
```
Comment (Oreoxmt) [inline tikv-control.md:123]:
```suggestion
  在此模式下，需要停止正在运行的 TiKV 实例。
```

File: tikv-control.md [hunk 9]
```diff
@@ -44,18 +44,18 @@ FLAGS:
     -V, --version                 Prints version information
 
 OPTIONS:
-        --ca-path <ca_path>              Set the CA certificate path
-        --cert-path <cert_path>          Set the certificate path
-        --config <config>                Set the config for rocksdb
-        --db <db>                        Set the rocksdb path
+        --ca-path <ca-path>              Set the CA certificate path
+        --cert-path <cert-path>          Set the certificate path
+        --config <config>                TiKV config path, by default it's <deploy-dir>/conf/tikv.toml
```
Comment (TomShawn) [inline tikv-control.md:49]:
```suggestion
        --config <config>                TiKV config path, by default it is <deploy-dir>/conf/tikv.toml
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**